pkg/core/xds/types.go

```diff
@@ -194,6 +194,7 @@ type ZoneIngressProxy struct {
 	TrafficRouteList *core_mesh.TrafficRouteResourceList
 	GatewayRoutes    *core_mesh.MeshGatewayRouteResourceList
 	MeshGateways     *core_mesh.MeshGatewayResourceList
+	PolicyResources  map[core_model.ResourceType]core_model.ResourceList
```

Comment:
I think we should either put `TrafficRouteList` into this `PolicyResources` map or create `MeshHTTPRouteList` field instead `PolicyResources` map

Reply:
the second option leads to an import cycle. I'll put `TrafficRoute` into the map.